### PR TITLE
Use dct:contributor instead of created_by & unify all attribution to ORCID

### LINKED ISSUE #278: Unify all attribution to ORCID

### LINKED ISSUE #281: Need to standardise created by annotations
Either oio:created_by or dc:creator. Opinions welcome. @nicolevasilevsky @sbello what do you use usually? HP/MP/Mondo?

### PR DESCRIPTION
**Pull request by anitacaron**
Fixes #281
Fixes #278

### CODE REVIEW

File: src/ontology/pato-edit.obo [hunk 1]
```diff
@@ -14804,7 +14804,7 @@ subset: mpath_slim
 subset: value_slim
 is_a: PATO:0002067 ! stratification
 relationship: is_opposite_of PATO:0002068 ! stratified
-created_by: George Gkoutos
+property_value: http://purl.org/dc/elements/1.1/contributor https://orcid.org/0000-0002-2061-091X
```
Comment (matentzn) [inline src/ontology/pato-edit.obo:14807]: Let's use `http://purl.org/dc/terms/contributor`. What you have is not wrong, but this is the recommended one.

Check: https://dublincore.org/specifications/dublin-core/dcmi-terms/#contributor

**Review by matentzn — CHANGES_REQUESTED** [review]
Can we also add the sparql check to look for unintended Annotation Properties in this PR?

**Review by shawntanzk — DISMISSED** [review]

File: src/ontology/pato-edit.obo (hunk 1 above)
Comment (matentzn) [reply src/ontology/pato-edit.obo]: Great!

**Comment by matentzn** [comment]
See comment above:

> Can we also add the sparql check to look for unintended Annotation Properties in this PR?

**Comment by matentzn** [comment]
Great! You still need to run `sh run.sh make update_repo` to apply the new test! Awesome thank you @anitacaron

**Review by matentzn — APPROVED** [review]
Great work! Ready for merging :)

**Review by shawntanzk — APPROVED** [review]

**Pull request merged**